src/sparseml/pytorch/optim/modifier_quantization.py

```diff
@@ -364,15 +367,17 @@ def _enable_module_qat(self, module: Module):
 
         # prepare each module / submodule for quantization
         qconfig = get_qat_qconfig()
-        for quant_module in self._modules_to_quantize:
+        for mod2quant in self._modules_to_quantize:
```

Comment:
please change out this variable name as well

Reply:
Fixed